lib/transaction_event_store_mongoid/repository.rb

```diff
@@ -121,7 +121,7 @@ def in_transaction?
     def build_event_model(event)
       {
         event_id:   event.event_id,
-        event_type: event.class,
+        event_type: event.class.to_s,
```

Comment:
@serch I think `class.name` would be better.

Reply:
@gingerhendrix done